### ISSUE TITLE
Unexpected Error when creating an IOT Hub device

### ISSUE THREAD

**Issue by etf5000** [Azure/azure-iot-cli-extension#434]
## Describe the bug

**Command Name**
`az iot hub device-identity create
Extension Name: azure-iot. Version: 0.10.16.`

**Errors:**
```
The command failed with an unexpected error. Here is the traceback:
'ItemPaged' object has no attribute 'advance_page'
Traceback (most recent call last):
  File "D:\a\1\s\build_scripts\windows\artifacts\cli\Lib\site-packages\knack/cli.py", line 231, in invoke
  File "D:\a\1\s\build_scripts\windows\artifacts\cli\Lib\site-packages\azure/cli/core/commands/__init__.py", line 657, in execute
  File "D:\a\1\s\build_scripts\windows\artifacts\cli\Lib\site-packages\azure/cli/core/commands/__init__.py", line 720, in _run_jobs_serially
  File "D:\a\1\s\build_scripts\windows\artifacts\cli\Lib\site-packages\azure/cli/core/commands/__init__.py", line 691, in _run_job
  File "D:\a\1\s\build_scripts\windows\artifacts\cli\Lib\site-packages\azure/cli/core/commands/__init__.py", line 328, in __call__
  File "D:\a\1\s\build_scripts\windows\artifacts\cli\Lib\site-packages\azure/cli/core/commands/command_operation.py", line 121, in handler
  File "C:\Users\Owner\.azure\cliextensions\azure-iot\azext_iot\operations\hub.py", line 165, in iot_device_create
    target = discovery.get_target(
  File "C:\Users\Owner\.azure\cliextensions\azure-iot\azext_iot\iothub\providers\discovery.py", line 173, in get_target
    target_policy = self.find_policy(
  File "C:\Users\Owner\.azure\cliextensions\azure-iot\azext_iot\iothub\providers\discovery.py", line 120, in find_policy
    policy_list = self.get_policies(hub_name=hub_name, rg=rg)
  File "C:\Users\Owner\.azure\cliextensions\azure-iot\azext_iot\iothub\providers\discovery.py", line 78, in get_policies
    policy_list.extend(policy_pager.advance_page())
AttributeError: 'ItemPaged' object has no attribute 'advance_page'

```

## To Reproduce:
Steps to reproduce the behavior. Note that argument values have been redacted, as they may contain sensitive information.

- _Put any pre-requisite steps here..._
- `az iot hub device-identity create --device-id {} --hub-name {} -g {}`

## Environment Summary
```
Windows-10-10.0.19041-SP0
Python 3.8.9
Installer: MSI

azure-cli 2.28.0

Extensions:
azure-iot 0.10.16

```

**Comment by c-ryan-k** [comment]
Thanks for the issue @etf5000 - we're currently rolling out a fix for this (#431) in a new version.
In the meantime, please use version `0.10.15` (we've rolled the extension index back to use this as the latest version):
`az extension remove --name azure-iot` and then
 `az extension add --name azure-iot` or use `az extension add --name azure-iot --version 0.10.15` to specify that version.
 
 I'll update this issue once the new version has been released.

**Comment by digimaun** [comment]
The latest released `0.10.17`, was published [6 days ago](https://github.com/Azure/azure-iot-cli-extension/issues/431#issuecomment-920492903) with the fix.
